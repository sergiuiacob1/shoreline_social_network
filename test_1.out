Distance between 1 and 1: 0Distance between 1 and 7: -1Distance between 1 and 6: 5