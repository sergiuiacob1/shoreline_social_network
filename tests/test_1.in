8
5
1 2
2 3
3 4
4 5
5 6
3
1 1
1 7
1 6