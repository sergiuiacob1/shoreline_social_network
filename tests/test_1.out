Distance between 1 and 3: -1
